values.yaml

```diff
@@ -218,6 +217,8 @@ zabbixproxy:
   extraVolumes: []
   # -- additional specifications to the zabbix proxy pod
   extraPodSpecs: {}
+  # -- extra volumeClaimTemplate for zabbixproxy statefulset
+  # extraVolumeClaimTemplate: []
```

Comment:
Hi @tomsozolins!

With my suggestion implemented in before file, you can to uncomment the new parameter.

Reply:
@aeciopires adjusted